source/index.rst

```diff
@@ -47,6 +47,7 @@ Table of Contents
    chapters/effects
    chapters/djing_with_mixxx
    chapters/preferences
+   chapters/import_from_serato
```

Comment:
Please remove this and amend the commit.

Reply:
Hello @Holzhaus can you give me sometime to fix this? I used ```git reflog``` and ```git reset head``` inappropriately and my working tree is messed up

Reply:
You can reset your working tree to the version you pushed here, by using

    git reset --hard origin/manual-2.2.x

(You should probably make a backup first by using `git branch -c mybackup`).

Next, you need to find out which commit introduced the unwanted change. By using `git log` and `git show`, you can easily check that it was this one: https://github.com/mixxxdj/manual/commit/41e0e66f10f736aa1df691e66f2e0e3c7c36c7b2#diff-c05e5dd7e57e7292e7fccc39bebe079e9a2062e0c9a3a57a4b6c608bd8d5669b

Since it's the last commit, you can just go ahead and remove that line, than run `git commit --amend`.

Reply:
Thank you so much for the help @Holzhaus I ran into some problems too while doing this ```git reset --hard origin/manual-2.2.x```. I resolved into updating the files manually since the initial reset didn't wipe everything away. please review. Thank you!